aws/resource_aws_appsync_graphql_api.go

```diff
@@ -230,6 +241,11 @@ func resourceAwsAppsyncGraphqlApiUpdate(d *schema.ResourceData, meta interface{}
 		return err
 	}
 
+	err = resourceAwsAppsyncSchemaPut(d.Id(), d, meta)
+	if err != nil {
+		return err
```

Comment:
We should return error context here for operators and code maintainers:

```suggestion
		return fmt.Errorf("error updating AppSync GraphQL API (%s) Schema: %s", d.Id(), err)
```